### PR TITLE
Update Readme logos and texts

### LINKED ISSUE #53: Logos are missing on the readme
I just spotted that the SIROP (logo) is missing in the project list in the README. That should be added.

### PR DESCRIPTION
**Pull request by Ludee**
Closes #53

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -2,21 +2,20 @@
 # Open Energy Family - Organisation
 
 This repository is created for discussions and decisions regarding the Open Energy Family.
-It is a collection of various information and tools that help to "set energy data free".
-The Open Energy Family is currently and on the long-run under development.
+It is a collection of various tools and information and that help working with energy related data. It is a collaborative community effort, everything is openly developed.
```
Comment (han-f) [inline README.md:5]: Suggest to write: The Open Energy Family (OEF) is a collection of various tools and information and that provide support in working with energy related data. Te OEF is a collaborative community effort and everything is openly developed.

File: README.md [hunk 2]
```diff
@@ -70,40 +68,56 @@ Several individuals across different research projects contribute to the develop
...
 
-﻿<a href="https://www.oeko.de/"><img src="https://reiner-lemoine-institut.de/wp-content/uploads/2018/02/oeko-institut-logo_transp.png" alt="Logo Öko-Institut" width="230px"></a> 
+﻿<a href="https://www.fortiss.org/"><img src="https://openenergy-platform.org/static/logos/partner/partner_logo_fortiss.png" alt="Logo fortiss" width="230px"></a> 
+
+﻿<a href="https://www.epe.ed.tum.de/ens/startseite/"><img src="https://www.epe.ed.tum.de/typo3/ext/tumtheme/Resources/Public/Icons/Org/tum-logo.png" alt="Logo TUM" width="170px"></a> 
 
 ## Licenses
 
-The Open Energy Family follows the ideal of Open Science. Thus all material (Code, Data, Artwork) will be licensed with suitable open licenses if possible.
+The Open Energy Family follows the ideal of Open Science. Thus all material (Code, Data, Artwork) will be licensed with suitable open licenses.
```
Comment (han-f) [inline README.md:106]: Suggest: The Open Energy Family follows the ideal of Open Science. Thus all material (Code, Data, Artwork) is licensed with a suitable open license.

File: README.md [hunk 3]
```diff
@@ -70,40 +68,56 @@ Several individuals across different research projects contribute to the develop
...
+﻿<a href="https://www.bmwk.de/Navigation/EN/Home/home.html"><img src="https://www.bmwk.de/SiteGlobals/BMWI/StyleBundles/Bilder/bmwi_logo_en.svg?__blob=normal&v=13" alt="Logo BMWK" width="200px"></a> 
 
 Coordinated by the Projektträger Jülich (PTJ). 
 
 ﻿<a href="https://www.ptj.de/"><img src="https://reiner-lemoine-institut.de/wp-content/uploads/2016/01/logo_PTJ.png" alt="Logo PTJ" width="160px"></a> 
- 
-## Contribute
-Anyone is invited to help develop the Open Energy Family. 
 
+Supported by the Reiner Lemoine Stiftung (RLS).
```
Comment (han-f) [inline README.md:118]: ? is this needed? what is this for`?

**Review by han-f — COMMENTED** [review]
Many thanks for this update, I added a few comments with suggestions. Please have a look.

File: README.md [hunk 4]
```diff
@@ -2,21 +2,20 @@
...
 Some family members are very well known and are in the foreground while others are rather small and inconspicuous. 
 Some have their own repository while some are directly attached to other members. 
-Together they make a strong family circle for working with open data and open science.
+Together, they form a strong family circle for Open Science in energy system research.
 
 <a href="https://openenergy-platform.org/"><img src="https://raw.githubusercontent.com/OpenEnergyPlatform/organisation/master/logo/OpenEnergyFamily_Logo_OpenEnergyPlatform_OEP.png" width="50px"></a>
 [`Open Energy Platform (OEP)`](https://openenergy-platform.org/)
 <a href="https://github.com/OpenEnergyPlatform/oeplatform"><img src="https://github.com/fluidicon.png" width="30px"></a><br>
-A web interface to access most of the modules, especially the community database.
+A web interface to access most of the modules, especially the community database and factsheets.
```
[the remark this answers is not in the document]
Comment (Ludee) [reply README.md:18]: Adding our definition of a "community database" would be great.

[the remark this answers is not in the document]
Comment (Ludee) [reply README.md:18]: > Everybody can contribute to the database by adding, editing, and reviewing information.

**Comment by Ludee** [comment]
Many thanks for the great feedback and suggestions @han-f 
I included everything already.

**Pull request merged**